spacy/morphology.pyx

```diff
@@ -37,7 +38,7 @@ cdef class Morphology:
         else:
             return shared_ptr[MorphAnalysisC]()
 
-    def _normalize_attr(self, attr_key, attr_value):
+    def _normalize_attr(self, attr_key : Union[int, str], attr_value : Union[int, str]) -> Tuple[str, Union[str, List[str]]]:
```

Comment:
We do return `None` if the input is of unexpected types, so then this should be `Optional`? Or should that code path raise instead of warn?